documentation/library-reference/source/lid.rst

```diff
@@ -273,17 +273,16 @@ and the arguments should be passed one per line and be one of the following:
 ``library.a``:
   Link against the specified static library.
 
-  .. note:: Note that this may cause a problem if you are using this to
-     link a static library that hasn't been built with ``-fPIC`` into a
-     shared library on the ``x86_64-linux`` platform.
+  .. note:: This may cause a problem if you are using this to link a static
+     library that hasn't been built with ``-fPIC`` into a shared library on the
+     ``x86_64-linux`` platform.
```

Comment:
Wonder why this calls out x86_64-linux, as aarch64-linux and x86_64-freebsd would be the same

Reply:
Done.